db/const.txt

```diff
@@ -668,9 +669,11 @@ RC_Demon	6
 RC_DemiHuman	7
 RC_Angel	8
 RC_Dragon	9
-RC_Boss	10
-RC_NonBoss	11
-RC_NonDemiHuman	12
+RC_Player	10
+RC_Boss	11
+RC_NonBoss	12
+RC_NonDemiHuman	13
```

Comment:
That wouldn't error out, since constant will be assumed as 0, 
Also we don't know why we have NonDemiHuman.. was it sure it was because player was considered as DemiHuman?